### ISSUE TITLE
help for bad result on test data

### ISSUE THREAD

**Issue by mdanesh90** [JaidedAI/EasyOCR#941]
Hi 
I train this model for my data in persian and english
but i have bad result after i use my weight with 90% accuracy in training on reader function and inference model. 
my result is like repeating some words not even close to right label. 
I try this on my train data also this can't recognize the label. 
I used easyocr before I  train the model and for that picture used in test and train and it could recognize but after I training on my data it can't.
So can anyone have idea of what is my wrong?

**Comment by EivindKjosbakken** [comment]
I would guess your dataset is not diverse enough, meaning the model gets overfitted to the custom data you are feeding it. In order to deal with this you need a large diverse dataset, which you can for example do with synthetic generation. I made an article in TowardsAI about that here if you are interested: [https://pub.towardsai.net/how-to-make-a-synthesized-dataset-to-fine-tune-your-ocr-3573f1a7e08b](https://pub.towardsai.net/how-to-make-a-synthesized-dataset-to-fine-tune-your-ocr-3573f1a7e08b). Also, I would test training less with your data (for example by lowering learning rate, or running fine-tuning for fewer epochs/iterations), as this can help prevent overfitting.
